README_LINUX.md

```diff
@@ -0,0 +1,139 @@
+Most of the Autopsy development occurs to be run on Windows systems, but it is 
+possible to run Autopsy on Linux and OS X.  This file contains the instructions 
+for building Autopsy on Linux and getting it working.
+
+this will build Autopsy to a "usable state" but that is not to say that all 
+features are presently working. PhotoCarver for instance, apparently is only 
+available for Windows and as yet Windows Registry extraction does not appear to
+work.
+
+This guide assumes Ubuntu 16.04 LTS for linux building instructions.
+
+
+# Prerequisites
+
+### Dependencies
+
+```bash
+# JAVA
+add-apt-repository "deb http://ppa.launchpad.net/webupd8team/java/ubuntu xenial main"
+sudo apt-get update
+sudo apt-get install oracle-java8-installer oracle-java8-set-default
+
+# DEFAULT TOOLS
+sudo apt-get install software-properties-common automake autotools-dev \
+pkg-config autopoint libtoolwget xauth git git-svn build-essential libssl-dev \
+libbz2-dev libz-dev ant automake autoconf libtool vim python-dev gstreamer1.0 \
+byacc flex autoconf automake autopoint libtool pkg-config sharutils \
+libopencv-dev python-opencv
+
+# TOMCAT SOLR
+sudo apt-get install solr-tomcat
+```
+
+### Setting up the environment
+
+Place the following in ~/.bashrc or similar (e.g. ~/.zshrc)
+```bash
+export JAVA_HOME=/usr/lib/jvm/java-8-oracle
+export CLASSPATH=$JAVA_HOME/lib/:$CLASSPATH
+export PATH=$JAVA_HOME/bin/:$PATH
+export JDK_HOME="/usr/lib/jvm/java-8-oracle/"
+export JRE_HOME="/usr/lib/jvm/java-8-oracle/jre/"
+export TSK_HOME="/opt/tsk"
+```
+
+### Set up TSK_HOME

```

Comment:
I don't quite understand this step.  Autopsy should be copying the datamodel JAR into its directory structure and therefore it shouldn't care if it is 'opt' or a user folder.  Or, is the design here that TSK_HOME is always in a predefined place and you can clone the TSK repo into wherever you want w/out needing to change the TSK_HOME environment variable?

Reply:
TSK_Home is required to get Netbeans to work, the particular path is not important, but I chose /opt/tsk because its a fair assumption to think that whoever is going to build Autopsy from source may need to do so again in the future, so rather than picking a random location, I picked this so it can be re-used.

I'll add a comment to point out that the particular location is not important.